payments-core/src/main/java/com/stripe/android/model/PaymentMethodOptionsParams.kt

```diff
@@ -80,16 +80,19 @@ sealed class PaymentMethodOptionsParams(
 
     @Parcelize
     data class Blik(
```

Comment:
You'll need to `@JvmOverloads constructor` these types of changes, otherwise you'll break binary compatibility. You can see the breakage in the .api file having removed the single argument constructor.